Octokit.Tests.Conventions/PreviewsTests.cs

```diff
@@ -70,7 +70,8 @@ public void NoStalePreviews()
                 // https://developer.github.com/v3/repos/commits/#get-a-single-commit
                 "application/vnd.github.v3.sha",
                 // https://developer.github.com/v3/activity/starring/#alternative-response-with-star-creation-timestamps
-                "application/vnd.github.v3.star+json"
+                "application/vnd.github.v3.star+json",
+                "application/vnd.github.v3.repository+json"
```

Comment:
Do you mind explaining to me the reason for this change?

Reply:
I added RepositoryContentMediaType to AcceptHeaders for passing repository media type in CheckTeamPermissionsForARepositoryWithCustomAcceptHeader. This caused this convention test to fail because now it expects "application/vnd.github.v3.repository+json" in defaultHeaders